doc/user_guide/References.ipynb

```diff
@@ -265,6 +265,92 @@
     ":::"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "ae3232e6-91dd-4a09-8301-b5a79ae42f8f",
+   "metadata": {},
+   "source": [
+    "## Assigning refs literally with `LiteralRef`\n",
+    "\n",
+    "Sometimes you don’t want Param to resolve a ref-like value on assignment—you want to store the ref object itself.\n",
+    "\n",
+    "Use `LiteralRef` to assign a ref as-is (verbatim), so you can forward it, serialize it, or resolve it later under different conditions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15985908-d821-41a6-a4e0-41a8a49d9aae",
+   "metadata": {},
+   "source": [
+    "\n",
+    "By default, assigning a ref-like value resolves and links it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2c80606-f60f-4534-b2da-608efd2c9ab8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Example(param.Parameterized):\n",
+    "    value = param.Parameter(allow_refs=True)\n",
```

Comment:
Just to avoid the possible confusion that this would only work with the base `Parameter`.

```suggestion
    "    value = param.String(allow_refs=True)\n",
```